app/javascript/components/CityTile.js

```diff
@@ -5,22 +5,21 @@ const CityTile = props => {
   if (props.banner.url != null ){
 
     return(
-      <div class="container">
+      <div className="cityContainer">
```

Comment:
I noticed that your class names and id's follow different conventions. 

IE, you have a `has-form` class and a `show-description` id but you use camel case here. I would advise standardizing the convention.